tidy3d/plugins/invdes/function.py

```diff
@@ -0,0 +1,78 @@
+# post processing function utilities
+
+import typing
+
+import tidy3d as td
+import tidy3d.plugins.adjoint as tda
+import jax.numpy as jnp
+
+
+class Utilities:
+    """Funcional utilities that help define postprocessing functions more simply in ``invdes``."""
+
+    @staticmethod
+    def make_array(arr: typing.Any) -> jnp.ndarray:
+        """Turn something into a ``jnp.ndarray``."""
+        if hasattr(arr, "values"):
+            return jnp.array(arr.values)
+        return jnp.array(arr)
+
+    @staticmethod
+    def get_amps(sim_data: tda.JaxSimulationData, monitor_name: str, **sel_kwargs) -> jnp.ndarray:
+        """Grab amplitudes from a ``ModeMonitorData`` and select out values."""
+
+        monitor_data = sim_data[monitor_name]
+
+        if not isinstance(monitor_data, td.ModeData):
+            raise ValueError("'grab_amps' only works with data from 'ModeMonitor's.")
```

Comment:
Function is called `get_amps`, I guess the error message should also refer to `get_amps` (not `grab_amps`).

Reply:
Ah, good catch, thanks. Fixed this and the field component one.